Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.6
+// swift-tools-version:5.7
```

Comment:
Tested the build and the lowest it will build on is `5.7` which reflects Xcode 14.0 mentioned in the 3.0.0 release